exercises/practice/leap/.meta/test_template.erb

```diff
@@ -0,0 +1,20 @@
+require 'minitest/autorun'
+require_relative 'leap'
+
+class Date
+  def leap?
+    raise "Implement this yourself instead of using Ruby's implementation."
```

Comment:
```suggestion
    raise "Write this functionality yourself instead of using Ruby's `leap?` method."
```

Reply:
Because "implement the implement you implement yourself, rather than the implement that Ruby implements" can be very confusing, if accurate and proper use, but I tend to avoid this word in multicultural and multilingual audiences.  

English is hard enough as it is, using a word with 4 senses when people tend to not take care or be clear which sense they are talking about can be doubly hard for those that do not have English as a first language.